otx/algorithms/classification/adapters/mmcls/models/backbones/mmov_backbone.py

```diff
@@ -0,0 +1,46 @@
+"""Module for the MMOVBackbone class."""
+
+from typing import Dict, List
+
+from mmcls.models.builder import BACKBONES
+
+from otx.mpa.modules.ov.graph.parsers.cls.cls_base_parser import cls_base_parser
+from otx.mpa.modules.ov.models.mmov_model import MMOVModel
+
+
+@BACKBONES.register_module()
+class MMOVBackbone(MMOVModel):
+    """MMOVBackbone class.
+
+    Args:
+        *args: positional arguments.
+        **kwargs: keyword arguments.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Is this `__init__` function necessary? how about removing it if you want to execute just parent class's function?